.angular-cli.json

```diff
@@ -19,9 +19,16 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.scss",
-        "assets/scss/main.scss"
+        "assets/scss/styles.scss"
       ],
+      "stylePreprocessorOptions": {
+        "includePaths": [
```

Comment:
this is cool -- I didn't realize this existed. I've been including them in a main scss file and pushing that in the styles array.